pkg/file/scripts/install_deb.sh

```diff
@@ -0,0 +1 @@
+apt-get install "${INSTALLER_PATH}"
```

Comment:
@dantecatalfamo @mna is it okay to use `apt-get` for deb packages? feels like if we use `dpkg` we'll be asking too much prep for the IT admin.

Reply:
We'll have to make sure the path is either an absolute or begins with `./` so we don't confuse apt. We should also use `-f` to install missing dependencies

Reply:
nice thanks! adding the `-f`. Should we do that here or leave it to the caller?

Reply:
Add it here, all it does is install missing packages and fix any dependency issues. If admins don't want that for some reason they can remove it

Reply:
@dantecatalfamo thanks, and sorry for not being clear, I meant for this part

> We'll have to make sure the path is either an absolute or begins with ./ so we don't confuse apt.

should we do any extra checks here?